pkg/local/sync.go

```diff
@@ -213,7 +213,7 @@ func (s *SyncManager) download(ctx context.Context, rootPath string, remote *uri
 	if err := fileutil.VerifyRelPath(strings.TrimPrefix(path, uri.PathSeparator), rootPath); err != nil {
 		return err
 	}
-	destination := fmt.Sprintf("%s%c%s", rootPath, os.PathSeparator, path)
+	destination := filepath.ToSlash(fmt.Sprintf("%s%s%s", rootPath, uri.PathSeparator, path))
```

Comment:
Is there any chance this will cause problems in some OS?

Reply:
This actually solves the issue with different OSs (ToSlash)